dao/src/main/java/org/thingsboard/server/dao/tenant/TenantServiceImpl.java

```diff
@@ -178,7 +178,7 @@ public void deleteTenant(TenantId tenantId) {
                 EntityType.TB_RESOURCE, EntityType.OTA_PACKAGE, EntityType.RPC, EntityType.QUEUE,
                 EntityType.NOTIFICATION_REQUEST, EntityType.NOTIFICATION_RULE, EntityType.NOTIFICATION_TEMPLATE,
                 EntityType.NOTIFICATION_TARGET, EntityType.QUEUE_STATS, EntityType.CUSTOMER,
-                EntityType.DOMAIN, EntityType.MOBILE_APP, EntityType.OAUTH2_CLIENT
+                EntityType.DOMAIN, EntityType.MOBILE_APP_BUNDLE, EntityType.MOBILE_APP, EntityType.OAUTH2_CLIENT
```

Comment:
Update `HousekeeperServiceTest#whenTenantIsDeleted_thenDeleteAllEntitiesAndCleanUpRelatedData` and test this case.